app/views/publisher_information/beta_capabilities.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :browser_title, t("publisher_information.beta_capabilities.title") %>
+<div class="govuk-grid-row">
+  <%= render partial: "publication_link_list", locals: { active: beta_capabilities_path } %>
```

Comment:
The 'active' name confused me for a while - can we have 'current_page' or something? 

Same with the name 'publication_link_list', which made me expect to see a 'list of links to publications'. Perhaps something like 'publication_info_nav'?

Reply:
DONE: Yep that improves readability.